### PR TITLE
[v10.3.x] Docs: Add annotated image and updated feature overview list

### PR DESCRIPTION
**Pull request by imatwawana**
Backport 47b42c9d4210bfbe274b082a2b1b4714426b19bd from #87833

### CODE REVIEW

File: docs/sources/panels-visualizations/panel-overview/index.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+[Data sources]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/connect-externally-hosted/data-sources"
+
+[ai]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/dashboards/manage-dashboards#set-up-generative-ai-features-for-dashboards"
+[ai]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/dashboards/manage-dashboards#set-up-generative-ai-features-for-dashboards"
+
+[panel links]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/dashboards/build-dashboards/manage-dashboard-links#panel-links"
+[panel links]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/dashboards/build-dashboards/manage-dashboard-links#panel-links"
+
+[tooltips]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/configure-tooltips"
+[tooltips]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/configure-tooltips"
```
Comment (jdbaldry) [inline docs/sources/panels-visualizations/panel-overview/index.md:132]: This 404s for me

**Review by jdbaldry — COMMENTED** [review]
Rest looks good but one link 404s

File: docs/sources/panels-visualizations/panel-overview/index.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+[Data sources]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/datasources"
+[Data sources]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/connect-externally-hosted/data-sources"
+
+[ai]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/dashboards/manage-dashboards#set-up-generative-ai-features-for-dashboards"
+[ai]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/dashboards/manage-dashboards#set-up-generative-ai-features-for-dashboards"
+
+[panel links]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/dashboards/build-dashboards/manage-dashboard-links#panel-links"
+[panel links]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/dashboards/build-dashboards/manage-dashboard-links#panel-links"
+
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/panel-overview/index.md:129]:
```suggestion
```

File: docs/sources/panels-visualizations/panel-overview/index.md (hunk 1 above)
Comment (imatwawana) [reply docs/sources/panels-visualizations/panel-overview/index.md]: Thanks for the catch!

**Review by jdbaldry — APPROVED** [review]

**Pull request merged**